io.github.rafostar.Clapper.Enhancers.Default.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "io.github.rafostar.Clapper.Enhancers.Default",
+    "runtime": "com.github.rafostar.Clapper",
+    "runtime-version": "stable",
+    "sdk": "org.freedesktop.Sdk//24.08",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": true,
+    "build-options": {
+        "env": {
+            "PIP_TARGET": "/app/extensions/clapper/enhancers/Default/python/site-packages"
+        }
+    },
+    "modules": [
+        {
+            "name": "python-modules",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install -v --root-user-action=ignore --no-deps *.whl"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/a4/97/73eadf12412173dc518897a2715693e6caa73d6550ddf0b7c47c6f1e7703/yt_dlp-2025.1.15-py3-none-any.whl",
+                    "sha256": "b8666b88e23c3fa5ee1e80920f4a9dfac7c405504a447214c0cf3d0c386edcfc",
+                    "x-checker-data": {
+                        "type": "pypi",
+                        "name": "yt-dlp",
+                        "packagetype": "bdist_wheel"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "clapper-enhancers",
+            "buildsystem": "meson",
+            "config-opts": [
+                "--buildtype=release",
+                "-Denhancersdir=/app/extensions/clapper/enhancers/Default/plugins",
+                "-Dauto_features=enabled"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Rafostar/clapper-enhancers.git",
+                    "tag": "0.8.2",
+                    "x-checker-data": {
+                        "type": "git",
+                        "tag-pattern": "^([\\d.]+)$"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "metainfo",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "io.github.rafostar.Clapper.Enhancers.Default.metainfo.xml"
+                }
+            ],
+            "build-commands": [
+                "install -Dm644 -t ${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml"
+            ]
+        }
```

Comment:
merge the sources with the above and add build-commands as `post-install`.

Reply:
I did that, but I also had to add `../` to file path. It seems `meson` build system runs inside build dir and not in project root.